packages/desktop/scripts/notarize.macos.js

```diff
@@ -19,7 +19,7 @@ exports.default = async () => {
 
     await notarize({
         appBundleId: 'org.iota.firefly',
-        appPath: path.resolve(__dirname, '../out/mac/Firefly.app'),
+        appPath: path.resolve(__dirname, '../out/mac/Firefly Beta.app'),
```

Comment:
Revert back before merge:
```suggestion
        appPath: path.resolve(__dirname, '../out/mac/Firefly.app'),
```